specs/slider/slider.md

```diff
@@ -0,0 +1,201 @@
+# slider + slider-label
+
+## Overview
+
+The `slider` component allows the user to move one or two `thumb` elements along a vertical or horizontal axis to select a value or range. `slider` is also considered an input field and must work within a `<form>` as other input fields do.
+
+### Background
+
+Just styling `<input type="range"/>` and using `<datalist>` for the tickmarks was considered but the standards aren't supported enough to allow for the level customization we want to provide.
+
+** see [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range)
+
+### Use Cases
+
+- *A customer using the component on a web page.*
+On a web page the customer drags the slider from min to max values ultimately setting the value of the component to a value constrained by step and min and max values.
+
+- *A developer building an app with the component and interacting through HTML/CSS/JavaScript.*
+A developer can inject labels on the slider as children, if they use <slider-label> elements those will be styled a specific way internally and the dev can override any part of that style they want. Alternatively the dev can put whatever they want in children and style as they wish.
+
+- *A designer customizing the component.*
+A designer can override any internal styling applied to the slotted labels, the background track, the slider track and the thumb(s) if appropriate.
+  
+### Features
+- Implement your own lower and upper thumb UI, default provided.
+- Implement your own background and slider track UI, default provided.
+- Developers can hook up to the `slider`'s `onValueChange` callback to be notified when the value has changed.

```

Comment:
This is nitpicky, but for the first two this reads like you are speaking to a developer personally and informally, while the third sentence simply says "Developers". Since I think the target audience here is the same can we align to one?

Reply:
yep will do